src/components/createEventPageComponents/CreateEvent.jsx

```diff
@@ -86,142 +88,157 @@ function CreateEvent() {
     }
 
     return (
-        <div className='m-0 box-border bg-bgc-silver'>
-            <div className='relative overflow-hidden w-full h-96'>
-                <Image
-                    src='/images/publicSpeak.jpg'
-                    alt='plant'
-                    className='w-full h-auto'
-                    width={1920}
-                    height={1080}
-                />
-                <div className='absolute inset-0 flex flex-col items-center justify-center text-white'>
-                    <div className='bg-black bg-opacity-50 p-8'>
-                        <Image
-                            className=' mx-auto my-auto'
-                            src='/images/HeroLogo.png'
-                            alt='logo'
-                            width={40}
-                            height={60}
-                        ></Image>
-                        <h1 className='text-4xl font-bold text-center mb-2 font-Montserrat text-white'>
-                            {t("createEvent.title")}
-                        </h1>
-                        <p className='text-lg font-Lora text-center text-white'>
-                            {t("createEvent.intro")}
-                        </p>
+        <>
+            <IndexPage title='Create an Event' />
```

Comment:
i kinda unerstand why you'd use the indexPage components in componenets that are meant for dynamic pages, but for static pages, i defently think it's prefarble to put them in pages